modules/users/tests/server/user.server.model.tests.js

```diff
@@ -436,17 +440,18 @@ describe('User Model Unit Tests:', function () {
     });
 
     it('should allow valid email address - "abc-def@abc.com"', function (done) {
-      var _user = new User(user);
+      var _user1 = new User(user1);
 
-      _user.email = 'abc-def@abc.com';
-      _user.save(function (err) {
+      _user1.email = 'abc-def@abc.com';
+      _user1.save(function (err) {
         if (!err) {
-          _user.remove(function (err_remove) {
+          _user1.remove(function (err_remove) {
+            should.not.exist(err);
```

Comment:
If `err` exists then it means the test failed and there's no user object that needs removal.
If `err` doesn't exist then it means the test succeeded and we need to
1. remove the user that we just added (to clean up the db for another test)
2. assert that `err` doesn't exist